src/app/services/fiddle.service.spec.ts

```diff
@@ -1,11 +1,30 @@
 import {FiddleService} from './fiddle.service';
+import {Http, HTTP_PROVIDERS, Response} from 'angular2/http';
+import {  
+    expect, 
+    it, 
+    describe,
+    injectAsync,
+    TestComponentBuilder,
+    beforeEachProvider
+} from 'angular2/testing';
 
 describe('fiddle.service', () => {
   let fiddle: FiddleService;
   
-  beforeEach(()=>{
-    fiddle = new FiddleService(); 
-  });
+  beforeEachProviders(() => [
+    HTTP_PROVIDERS, 
+    FiddleService
+  ]);
+  
+  beforeEach(inject([FiddleService], s => {
```

Comment:
This is now what is breaking it. Seems like the same principle as referenced here:
https://medium.com/google-developer-experts/angular-2-unit-testing-with-jasmine-defe20421584#.d5wn531r2